test/unit/gateways/jetpay_v2_test.rb

```diff
@@ -19,6 +19,28 @@ def setup
     }
   end
 
+  def test_default_api_version
```

Comment:
My suggestion for a simpler test would just be to update `test_successful_purchase` to be something like we [this SecurePayAu test](https://github.com/activemerchant/active_merchant/blob/master/test/unit/gateways/secure_pay_au_test.rb#L52) you would just look for JetPay Version